sr/comp/cli/show_match_scores.py

```diff
@@ -0,0 +1,167 @@
+__description__ = "Show the game and league points achieved for each match"
+
+from typing import Dict, List, Union, NamedTuple
+from sr.comp.types import TLA, GamePoints, ArenaName, MatchNumber
+from sr.comp.scores import LeaguePosition
+
+
+class MatchCorner(NamedTuple):
+    tla: TLA
+    ranking: Union[LeaguePosition, str]  # to allow "???" to be used for unknown scores
+    game: Union[GamePoints, str]
+    league: Union[int, str]
```

Comment:
I'd suggest expressing the unknown case as `None` and force the formatting layer to handle it. That keeps the presentation layer separate from the data layer.
Alternatively, would it make sense to introduce another type so that we have a `ScoredMatch` and a `FutureMatch`? That means that the type system disallows cases like a match having league points but no game points.

Reply:
I've moved to using `None` in 14e0af3d04629926b57f751aedcc75eb88e8cddf